client/components/product.js

```diff
@@ -1,28 +1,50 @@
-import React from 'react';
+import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import {withRouter} from 'react-router-dom';
 
-export const productComponent = (props) => {
-  const product = props.products.find(prod => Number(prod.id) === Number(props.match.params.id));
+export class productComponent extends Component {
 
-  return (
-    <div>
-      <div>
-        <h1>{product && product.name}</h1>
-        <h3>Price: {product && product.price}</h3>
-        <img src={product && product.imageUrl} />
-        <br />
-        <p>Description: {product && product.description}</p>
-        <button>Buy</button>
-      </div>
-      <hr />
+  render () {
+
+    const product = this.props.products.find(prod => Number(prod.id) === Number(this.props.match.params.id));
+
+    return (
       <div>
-        <h2>Reviews</h2>
+        <div>
+          <h1>{product && product.name}</h1>
+          <h3>Price: {product && product.price}</h3>
+          <img src={product && product.imageUrl} />
+          <br />
+          <p>Description: {product && product.description}</p>
+          <form>
+            <select name="quantity">
+              <option>1</option>
+              <option>2</option>
+              <option>3</option>
+              <option>4</option>
+              <option>5</option>
+              <option>6</option>
+              <option>7</option>
+              <option>8</option>
+              <option>9</option>
+              <option>10</option>
```

Comment:
There's probably a better way to do this using JSX. Perhaps we could create an array with values 1-10.